contribution/vagrant/Vagrantfile

```diff
@@ -35,4 +43,9 @@ Vagrant.configure("2") do |config|
 
   # Initialize Kubernetes
   config.vm.provision :shell, path: kubearmor_home + "/contribution/self-managed-k8s/k8s/initialize_kubernetes.sh"
+
+  # Reboot to use new kernel
+  if ENV['NETNEXT'] == "1" then
+	  config.vm.provision :reload
```

Comment:
Yes, reloading is necessary because we are updating the kernel and we need to switch to the new one.

Reloading (even if you do vagrant halt/up) is causing ownership loss if you have an old VirtualBox-guest. If you try the `NETNEXT=1 vagrant plugin install vagrant-vbguest` this will update the VirtualBox-guest image and things work after that. The other option is for the user to use sudo when running kubearmor.

There is no other easy way to solve this problem. Using NFS to mount the synced_folder would be another option but it requires some other configuration.

Reply:
I have updated the documentation to explain these points.

Reply:
This looks great. One other way would be to declare the needed plugins inside the Vagrantfile.

```diff
@@ -9,6 +9,7 @@ else
 end
 
 Vagrant.configure("2") do |config|
+  config.vagrant.plugins = ["vagrant-vbguest", "vagrant-reload"]
   config.vm.define VM_NAME do |cfg|
```

It will ask one to install plugins locally and to run the requested command again.

```bash
❯ NETNEXT=1 vagrant up
Vagrant has detected project local plugins configured for this
project which are not installed.

  vagrant-vbguest
Install local plugins (Y/N) [N]: y
Installing the 'vagrant-vbguest' plugin. This can take a few minutes...
Fetching micromachine-3.0.0.gem
Fetching vagrant-vbguest-0.30.0.gem
Installed the plugin 'vagrant-vbguest (0.30.0)'!


Vagrant has completed installing local plugins for the current Vagrant
project directory. Please run the requested command again.
```

Reply:
This is even better. Will try to handle this as part of another PR. Thanks